tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -496,12 +496,13 @@ ecmp/inner_hashing/test_wr_inner_hashing_lag.py:
 
 ecmp/test_ecmp_sai_value.py:
   skip:
-    reason: "Only support Broadcom T1/T0 topology with 20230531.12 or later image, 7050cx3 T1 doesn't enable this feature"
+    reason: "Only support Broadcom T1/T0 topology with 20230531 image, 7050cx3 T1 doesn't enable this feature"
     conditions_logical_operator: or
     conditions:
       - "topo_type not in ['t1', 't0']"
       - "asic_type not in ['broadcom']"
-      - "release in ['201911', '202012', '202205', '202211', 'master', 'internal']"
+      - "release in ['201911', '202012', '202205', '202211', '202311', 'master']"
+      - "'internal' in build_version"
```

Comment:
All branches like `internal-202012`, `internal-202305` will be skipped as well. Is this expected?

Reply:
@wangxin Yes, it should be skipped, it can only run on 202305 released image.

Reply:
@ZhaohuiS  for the 202311, should we use xfail?